web/components/navbar/index.js

```diff
@@ -66,11 +68,11 @@ class Navbar extends React.Component {
     const { displayType, buttonType } = this.props;
     const button = getButton(buttonType);
     const linkElements = [
-      <Link to="/"><b>About</b></Link>,
-      <Link to="/"><b>Stories</b></Link>,
-      <Link to="/"><b>FAQ</b></Link>,
-      <Link to="/"><b>Sponsors</b></Link>,
-      <Link to="/"><b>2018</b></Link>,
+      <Link to={{ pathname: '/', hash: SECTION.ABOUT }}><b>About</b></Link>,
+      <Link to={{ pathname: '/', hash: SECTION.STORIES }}><b>Stories</b></Link>,
+      <Link to={{ pathname: '/', hash: SECTION.FAQ }}><b>FAQ</b></Link>,
+      <Link to={{ pathname: '/', hash: SECTION.SPONSORS }}><b>Sponsors</b></Link>,
+      <a href="http://nwhacks.github.io/nwhacks2018_static"><b>2018</b></a>,
```

Comment:
Can we add `target=_blank` to open this page in a new tab ? Since it’s an external link